CharacterCreation/02_Classes.txt

```diff
@@ -465,8 +386,10 @@ Feats:
 	----------------------
 	Protective Awareness - Requires a minimum Reflex of 24
 	Assiduously observant, you stay aware of any attacks made against you or allies. 
-	Whenever an enemy attempts to make an opportunity attack on you or any allies within 
-	eye-sight, you first gain an attack on them.
+	Whenever an enemy (that you can see) attempts to make an OPPORTUNITY attack on you
+	or any allies, within your guns maximum range, you first gain an attack on them.
+	The first time this ability is triggered on a given turn, it uses your next action.
+	(But the ability only uses 1 action, even if triggered multiple times -before or during- one of your turns).
```

Comment:
Is Opportunity Attack defined somewhere?

Reply:
It is, actually. Only they are currently called "Penalty attacks." https://github.com/bleehu/Compound_X/blob/play/Basic%20Rules.txt#L427

Reply:
We should define it better in the glossary

Reply:
BTW, do you think this is a good change? (I'm guessing you do, since you didn't say anything about it, but verification is nice)